clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -346,8 +346,16 @@ private void cancelInFlightRequests(String nodeId, long now, Collection<ClientRe
             }
 
             if (!request.isInternalRequest) {
-                if (responses != null)
-                    responses.add(request.disconnected(now, null));
+                if (responses != null) {
+                    ClientResponse disconnected;
```

Comment:
nit: The variable name is not accurate anymore.

Reply:
This code has been refactored and there's no longer a new local variable set.